CRM/Core/BAO/EntityTag.php

```diff
@@ -483,4 +483,40 @@ public static function buildOptions($fieldName, $context = NULL, $props = []) {
     return $options;
   }
 
+  /**
+   * This function deletes entity tags when a related entity is called.
+   *
+   * It is registered as a listener in \Civi\Core\Container::createEventDispatcher
+   *
+   * Ideally we will later nuance it so it's only called on delete....
```

Comment:
That's done :)

Reply:
@colemanw good point - I've removed the comment - is this merge-on-pass now?